### PR TITLE
Implement strides for the `slice` operator

### PR DESCRIPTION
**Pull request by dominiklohmann**
This adds an option to add strides in the `slice` operator, and adds a `reverse` as a shorthand notation for `slice ::-1`.

### CODE REVIEW

File: web/docs/operators/slice.md [hunk 1]
```diff
@@ -62,3 +66,15 @@ Skip the first and the last event:
 ```
 slice 1:-1
 ```
+
+Return every second event from the tenth:
```
Comment (mavam) [inline web/docs/operators/slice.md:70]:
```suggestion
Return every second event starting from the tenth:
```

File: web/docs/operators/slice.md [hunk 2]
```diff
@@ -62,3 +66,15 @@ Skip the first and the last event:
...
+
+```
+slice 10::2
```
Comment (mavam) [inline web/docs/operators/slice.md:73]: @jachris will love you for making this look like an IPv6 literal.

File: web/docs/operators/reverse.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+sidebar_custom_props:
+  operator:
+    transformation: true
+---
+
+# reverse
+
+Reverse the event order.
```
Comment (mavam) [inline web/docs/operators/reverse.md:9]:
```suggestion
Reverses the event order.
```

**Review by mavam — COMMENTED** [review]
Didn't check the implementation, but the scaffolding makes sense.

**Review by jachris — APPROVED** [review]

**Pull request merged**